x/storage/keeper/keeper.go

```diff
@@ -1039,6 +1062,9 @@ func (k Keeper) ForceDeleteObject(ctx sdk.Context, objectId sdkmath.Uint) error
 			ctx.Logger().Error("unlock store fee error", "err", err)
 			return err
 		}
+		if ctx.IsUpgraded(upgradetypes.Pawnee) {
+			k.DecreaseCreatedObjectCount(ctx, bucketInfo.Id)
+		}
 	} else if objectStatus == types.OBJECT_STATUS_SEALED {
 		internalBucketInfo := k.MustGetInternalBucketInfo(ctx, bucketInfo.Id)
 		err := k.UnChargeObjectStoreFee(ctx, bucketInfo, internalBucketInfo, objectInfo)
```

Comment:
if there is object being updated,  update the payment addr is not allowed?

Reply:
added related codes, please check.